README.md

```diff
@@ -1,2 +1,16 @@
+
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+```
+for i (initialize i to 0, run this for loop 'WordLength' times, increment 'i' each run)
+	for j (init j to 0, run this for loop 'WordLength' times, increment 'j' each run)
+		if letter i of GuessedWord matches letter j of SecretWord
+			check the matching letter is in the same place by checking the integer i and j are the same value
+				if it is, increment the Score
+
+check if the player won, Score = WordLength?
+	GameWon = True
+else
+	print "Access denied. Likeness = 'Score' out of WordLength"
+```
+b142d3625f17585b6e86a858fbfb15b860cbfe67
```

Comment:
Flowchart:
- Most of the flowchart is actually just a linear series of steps -- this is not the kind of information that a flowchart is well-suited to representing
- There is too much specific detail about presentation etc -- flowcharts at best should be focussing on the control flow
- Could be clearer on how the number of guesses is tracked -- show a variable being initialised, decremented on each guess, and checked against zero
- Good reference to the pseudocode -- no need to duplicate information between the two